wolfssl/wolfcrypt/port/Espressif/esp32-crypt.h

```diff
@@ -89,11 +89,13 @@ uint64_t  wc_esp32elapsedTime();
 
 /* RAW hash function APIs are not implemented with esp32 hardware acceleration*/
 #define WOLFSSL_NO_HASH_RAW
+#define SHA_CTX DUMMY
```

Comment:
This can be solved a few ways. But this workaround is okay, however I would suggest a better name as "DUMMY" may also conflict. The ESP32 code that reports this is here:
https://github.com/espressif/arduino-esp32/blob/master/tools/sdk/include/esp32/rom/sha.h#L35

I would suggest using "#define SHA_CTX ETS_SHAContext`

Reply:
Thanks, I used your suggestion.